config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  devise_for :users
+  resources :users
```

Comment:
It's a good practice to limit routes only to the ones we implement - in that case, it's `only: :index`.